data/core/common.lua

```diff
@@ -869,4 +869,69 @@ function common.rm(path, recursively)
 end
 
 
+---Open the given resource using the system launcher.
+---The resource can be an url, file or directory in most cases...
+---@param resource string
+---@return boolean success
+function common.open_in_system(resource)
+  -- Detect platforms
+  local function detect_platform()
+    if PLATFORM:lower():find("unknown", 1, true) then
```

Comment:
In my port I set the platform like it can be seen at 
https://git.walkero.gr/walkero/lite-xl/src/branch/amiga2.1/src/main.c#L133-L136

Reply:
Just in case this is the code that sets the PLATFORM global variable: https://git.walkero.gr/walkero/lite-xl/src/branch/amiga2.1/src/main.c#L237-L239 

I will assume it is properly set to AmigaOS

Reply:
It is set to "AmigaOS 4" and "MorphOS" respectively